ODIN_II/SRC/subtractions.cpp

```diff
@@ -449,6 +459,11 @@ void split_adder_for_sub(nnode_t* nodeo, int a, int b, int sizea, int sizeb, int
                 if (i == count - 1 && flag == 1) {
                     /* If the input pin of not gate connects to gnd, replacing the input pin and the not gate with vcc;
                      * if the input pin of not gate connects to vcc, replacing the input pin and the not gate with gnd.*/
+                    /* connecting untouched nets in the netlist creation to the pad node */
```

Comment:
Should delete the duplicate comment (repeated line).

Reply:
So sorry, I uploaded the code quickly to see whether there is any failure or not.